app/src/main/AndroidManifest.xml

```diff
@@ -12,6 +12,7 @@
     android:roundIcon="@mipmap/ic_launcher_round"
     android:supportsRtl="true"
     android:theme="@style/OppiaTheme">
+    <meta-data android:name="expiration_date" android:value="2020-09-01" />
```

Comment:
Functionality to add: disabled flag rather than only relying on the date (so that it can always be disabled for developers by default).

Reply:
should the value be hardcoded in manifest file? anyone can read these values